src/nvim/testdir/test100.in

```diff
@@ -25,8 +25,10 @@ STARTTEST
 :call FillBuffer()
 :call feedkeys(":earlier 10\n", 't')
 :call UndoLevel()
+:set ft=unix
```

Comment:
@splinterofchaos in the original patch `ff` but not `ft` was setted. I found that by trying to merge 7.4.500. Was that difference a mistake or I should also use `ft` instead of `ff`?